Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM matchoffice/rubies:2.3.1
```

Comment:
Shouldn't we be pulling the trenpixster/elixir image up here? Or are we just using the Erlang Solutions package to provide Elixir?